core/templates/dev/head/galleries/gallery.html

```diff
@@ -78,6 +78,33 @@
             </div>
           </div>
           <activity-tiles-infinity-grid></activity-tiles-infinity-grid>
+
+          <div infinite-scroll="showMoreExplorations()" infinite-scroll-distance="1"
+               infinite-scroll-disabled="finishedLoadingPage || pageLoaderIsBusy">
+            <collection-summary-tile ng-repeat="collection in allCollectionsInOrder track by $index"
+                                     collection-id="collection.id"
+                                     collection-title="collection.title"
+                                     last-updated-msec="collection.last_updated_msec"
+                                     num-explorations="collection.num_explorations"
+                                     objective="collection.objective"
+                                     thumbnail-icon-url="collection.thumbnail_icon_url"
+                                     thumbnail-bg-color="collection.thumbnail_bg_color">
+            </collection-summary-tile>
+            <exploration-summary-tile ng-repeat="exp in allExplorationsInOrder track by $index"
```

Comment:
Wasn't aware that the snippet had been replaced by `activity-tiles-infinity-grid` directive. Removed.